apps/site/src/lib/components/ValidatingForm/ValidatingForm.tsx

```diff
@@ -25,10 +29,15 @@ function ValidatingForm(props: PropsWithChildren<FormProps>) {
 		<form
 			onSubmit={handleSubmit}
 			noValidate // use custom validation feedback
-			className={validated ? styles.validated : styles.notYetValidated}
+			className={`bg-white p-5 md:p-10 rounded-2xl mx-5 text-black ${
+				validated ? styles.validated : styles.notYetValidated
+			}`}
 			// validated={validated}
 			{...props}
-		/>
+		>
+			{props.children}
```

Comment:
Thought: we are already spreading props, so providing children to the element might be in conflict.
We can extract the children separately and provide the rest of the props.
```tsx
const { children, ...rest } = props;
<form {...rest}>
	{children}
	<button ...>
</form>
```